src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -63,6 +65,12 @@
                 .withTags(VALID_TAG_AVERAGE, VALID_TAG_EXPERIENCED).build();
     }
 
+    // ModuleClass related constants
+
+    public static final String VALID_NAME_CS2103T_TUTORIAL = "CS2103T Tutorial";
+    public static final String VALID_NAME_CS2101_TUTORIAL = "CS2101 Tutorial";
+    public static final String VALID_NAME_CS2100_TUTORIAL = "CS2100 Tutorial";
+
     /**
      * Executes the given {@code command}, confirms that <br>
      * - the returned {@link CommandResult} matches {@code expectedCommandResult} <br>
```

Comment:
Need full stop for this comment

Reply:
As these informal list points are not complete sentences and are fragments, I would be more inclined to leave the full stop out?

Reply:
Ok